src/software/geom/polygon.cpp

```diff
@@ -49,76 +49,6 @@ Point Polygon::centroid() const
     return Point((Vector(x_centre, y_centre) / (3 * signed_area)));
 }
 
-Polygon Polygon::expand(const Vector& expansion_vector) const
-{
-    // ASCII art showing an expanded Polygon
-    //
-    // Original Polygon:
-    //
-    //            B---------C
-    //            |         |
-    //            |         |
-    //            A---------D
-    //
-    //                 |
-    //                 V
-    //          expansion vector
-    //
-    // Expanded Polygon (A and D are shifted down):
-    //
-    //            B---------C
-    //            |         |
-    //            |         |
-    //            |         |
-    //            A'--------D'
-    //
```

Comment:
Why do we not want to keep this functionality? It seems to be giving more control over the expansion compared to `Polygon::expand(double)`.

Reply:
it's confusing to use and not well used (single direction expands was only used in one place)